### PR TITLE
Less hardcoded stuff

### PR DESCRIPTION
**Pull request by gcantoni**

### CODE REVIEW

File: appintro/src/main/res/layout/appintro_intro_layout2.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
             android:id="@+id/bottom_separator"
             android:layout_width="match_parent"
             android:layout_height="1px"
-            android:background="#55000000"
+            android:background="@color/textBg"
```
Comment (cortinico) [inline appintro/src/main/res/layout/appintro_intro_layout2.xml]: So this `<TextView>` can be removed entirely in this layout file.
Looks like the `bottom_separator` is used only on `layout` and not in `layout2`.
Can I ask you to apply this fix on the other layout file: `appintro/src/main/res/layout/appintro_intro_layout.xml`?

File: appintro/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
     <color name="appintro_default_background_color">#222222</color>
+    <color name="textBg">#55000000</color>
```
Comment (cortinico) [inline appintro/src/main/res/values/colors.xml]: Please rename to `appintro_default_separator_color` for consistency.

File: appintro/src/main/res/layout/appintro_intro_layout2.xml (hunk 1 above)
Comment (gcantoni) [reply appintro/src/main/res/layout/appintro_intro_layout2.xml]: What do you mean with applying this fix to the other layout file ?

Should i make non-hardcoded the color there too or delete the TextView?

Comment (cortinico) [reply appintro/src/main/res/layout/appintro_intro_layout2.xml]: > What do you mean with applying this fix to the other layout file ?

Sorry so, I'd:
* Remove the `<TextView>` from the `_layout2.xml` file
* Apply the fix (don't hardcode the color) from the `_layout.xml`

**Comment by Anutha02** [comment]
The issue this pull request solves is solved in #647 along with a lot others... So this PR is redundant...

**Pull request closed without merging**